packages/gatsby/src/schema/data-tree-utils.js

```diff
@@ -88,14 +97,15 @@ const extractFromEntries = (entries, selector, key = null) => {
   const values = entries.map(entry => entry.value)
 
   const exampleValue = entriesOfUniqueType[0].value
+
   if (isScalar(exampleValue)) {
     return getExampleScalarFromArray(values)
   } else if (_.isObject(exampleValue)) {
-    if (_.isArray(exampleValue)) {
-      const concatanedItems = _.concat(...values)
+    if (Array.isArray(exampleValue)) {
+      const concatanedItems = [].concat(...values)
       // Linked node arrays don't get reduced further as we
       // want to preserve all the linked node types.
-      if (_.includes(key, `___NODE`)) {
+      if (key.includes(`___NODE`)) {
```

Comment:
This may not be supported in node 4, do we use Babel polyfill tho? Should be fine if so

Reply:
`includes` is not supported by node 4. But [we do use babel-polyfill](https://github.com/gatsbyjs/gatsby/blob/66ef48924dc695c0d9b40282ed295252fc5179f4/packages/gatsby-cli/src/index.js#L5), so this should be ok.

Reply:
We more-or-less stopped supporting node 4 in v1 anyways since we broke it accidentally for several months and no one complained 🤷‍♂️